packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -1,46 +1,534 @@
-/*!
- * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
- */
+/* globals window */
```

Comment:
Remove?

Reply:
@mattnorris  Metrics needs some system information like `window.location.hostname ` and `window.navigator.platform` what the best way to get it rather than using the window object in the actual file .